COLLADAStreamWriter/include/COLLADASWConstants.h

```diff
@@ -202,6 +202,8 @@ namespace COLLADASW
         static const String CSW_ELEMENT_PASS;
         static const String CSW_ELEMENT_PH;
         static const String CSW_ELEMENT_PHONG;
+		static const String CSW_ELEMENT_PLUGIN_VERSION;
```

Comment:
In ColladaMayaFileTranslator.cpp , line 85
This string will be used in the info panel of the plugin manager
When you open it you will have something like that: 
1.3.0.developer version__sha1__c41992
if it is build with jenkins, it will change "developer version" 

the code to save the string into the .DAE is  in addVersionNumber (COLLADASWAsset.cpp)
Extra tag are added with CURRENT_REVISION and CURRENT_SHA1

```javascript
<asset>
    <extra>
      <technique profile="OpenCOLLADAMaya">
        <pluginVersion>developer version</pluginVersion>
        <sha1>74254ae</sha1>
      </technique>
    </extra>
 </asset>
```